docs/pages/guides/monorepos.md

```diff
@@ -92,10 +93,80 @@ config.resolver.nodeModulesPaths = [
   path.resolve(projectRoot, 'node_modules'),
   path.resolve(workspaceRoot, 'node_modules'),
 ];
+// 3. Force Metro to resolve (sub)dependencies only from the `nodeModulesPaths`
+config.resolver.disableHierarchicalLookup = true;
 
 module.exports = config;
 ```
 
+<Collapsible summary="1. Why do we need to watch all files with the monorepo?">
+
+This option enables the app to refresh or reload when you edit imported files outside the app folder. It's important for libraries within your monorepo that are imported into your app. Without this setting, you must manually reload your app and possibly clear the Metro cache to see the changed library files.
+
+If your monorepo becomes really big, watching all files within the monorepo might be slow. If you want to speed that up, you can (dynamically) watch the relevant paths to minimize the watchers. E.g.
```

Comment:
```suggestion
As your monorepo increases in size, watching all files within the monorepo becomes slower. If you want to speed that it, you can (dynamically) watch the relevant paths to minimize the watchers. E.g.
```

Reply:
`If you want to speed that it` doesn't sound right

Reply:
```suggestion
As your monorepo increases in size, watching all files within the monorepo becomes slower. If you want to speed that up, you can (dynamically) watch the relevant paths to minimize the watchers. E.g.
```